src/Microsoft.VisualStudio.Services.Agent/EnvVariableParts.cs

```diff
@@ -0,0 +1,11 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+namespace Microsoft.VisualStudio.Services.Agent
+{
+    public class EnvVariableParts
+    {
+        public string Prefix { get; set; }
```

Comment:
>Prefix [](http://example.com/codeflow?start=22&length=6)

[suggestion/nonblocking]
if these don't need to be changeable, better to set to read-only and initialize in constructor

Reply:
Good point. Fixed, thanks!